addons/l10n_sa/data/account_tax_report_translation_data.xml

```diff
@@ -0,0 +1,265 @@
+<odoo>
+    <data noupdate="1">
+        <record id="l10n_sa_tax_report_vat_filing_ar" model="ir.translation">
```

Comment:
Why ? Aren't .po enough ? No other l10n is doing this.

Reply:
I removed the file and added the content to a po file